src/js/utils/menu-utils.jsx

```diff
@@ -5,6 +5,7 @@ export const checkActiveSection = ({
   menuUrls,
   path,
   params,
+  menuSectionsUrlParts,
```

Comment:
If you are passing this here, then this entire util should be reworked to be based on this config and to get it working in the same way as on the grails side